tests/test_fips.py

```diff
@@ -350,3 +367,23 @@ def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
     c.check_output(
         'certutil -R -k rsa -g 2048 -s "CN=Daniel Duesentrieb3,O=Example Corp,L=Mountain View,ST=California,C=DE" -d "${PWD}/nssdb" -o cert9.cer -f password.txt -z seedfile.dat',
     )
+
+
+@pytest.mark.parametrize(
+    "container",
+    FIPS_OPENJDK_IMAGES,
+    indirect=True,
+)
+def test_openjdk_cert(container: ContainerData) -> None:
```

Comment:
Some documentation of of what you're doing here, especially of the java file would be helpful

Reply:
Added a comment

Reply:
:heavy_check_mark: